wallet/src/main/java/com/android/identity_credential/wallet/Util.kt

```diff
@@ -0,0 +1,32 @@
+package com.android.identity_credential.wallet
+
+import android.graphics.Bitmap
+import android.graphics.Color
+import com.google.zxing.BarcodeFormat
+import com.google.zxing.MultiFormatWriter
+import com.google.zxing.WriterException
+import com.google.zxing.common.BitMatrix
+
+fun encodeQRCodeAsBitmap(str: String): Bitmap {
+    val width = 800
+    val result: BitMatrix = try {
+        MultiFormatWriter().encode(
+            str,
+            BarcodeFormat.QR_CODE, width, width, null
+        )
+    } catch (e: WriterException) {
+        throw java.lang.IllegalArgumentException(e)
+    }
+    val w = result.width
+    val h = result.height
+    val pixels = IntArray(w * h)
+    for (y in 0 until h) {
+        val offset = y * w
+        for (x in 0 until w) {
+            pixels[offset + x] = if (result[x, y]) Color.BLACK else Color.WHITE
+        }
+    }
+    val bitmap = Bitmap.createBitmap(w, h, Bitmap.Config.ARGB_8888)
+    bitmap.setPixels(pixels, 0, width, 0, 0, w, h)
+    return bitmap
+}
```

Comment:
I'm not a big fan of `Util` files, they tend to grow beyond control as is the case with `Util.java` in the `identity` library. Just move this into `QrEngagementScreen.kt`, I doubt we'll need it elsewhere.